index.js

```diff
@@ -53,7 +53,7 @@ function listFiles(root, out) {
     if (err) return out.emit('error', 'Failed to pack archive: ' + err);
 
     // npm logs created filename on stdout
-    var tarFile = path.join(process.cwd(), stdout.trim());
+    var tarFile = path.join(process.cwd(), stdout.trim().split(/\s+/).pop());
```

Comment:
Wondering if it would be better to split on `'\n'` since the file will be on it's own line.
Also that would cover cases in which the filename has a white space (in case that ever happens).